automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/TestData.scala

```diff
@@ -1,35 +1,37 @@
 package org.broadinstitute.dsde.firecloud.fixture
 
-object TestData {
+import org.broadinstitute.dsde.workbench.service.Orchestration
 
-  /* PARTICIPANTS AND PARTICIPANT SETS*/
-  object SingleParticipant {
-    val participantEntity = "entity:participant_id\nparticipant1"
-    val entityId = "participant1"
-  }
+case class TestData(prefix: String = "participant") {
 
+  val entityId = Orchestration.randomIdWithPrefix("participant")
+  val participantEntity = s"entity:participant_id\n$entityId"
+  private val samplePrefix = Orchestration.randomIdWithPrefix("sample")
 
-  /* SAMPLES AND SAMPLE SETS */
-  object HundredAndOneSampleSet {
-    val entityId = "sampleSet101"
-    val sampleSetCreation = "entity:sample_set_id\n" +
-      "sampleSet101"
-    private val samplePrefix = "sample"
-    val participantId = SingleParticipant.entityId
-    val samples = createSamples(samplePrefix, participantId, 101)
-    val sampleSetMembership = createSampleSet(entityId, samplePrefix, 101)
+  def hundreAndOneSet: HundredAndOneSampleSet = {
+    new HundredAndOneSampleSet()
   }
 
-  private def createSamples(samplePrefix: String, participantId: String, numberOfSamples: Int): String = {
-    "entity:sample_id\tparticipant_id\n" +
-      (0 until numberOfSamples).map(i => s"$samplePrefix$i\t$participantId\n").mkString.stripSuffix("\n")
+  /* SAMPLES AND SAMPLE SETS */
+  class HundredAndOneSampleSet() {
```

Comment:
this can also be an `object`

Reply:
`object` are avoided because tests and test suites are running in parallel.